custom/icds_reports/utils/data_pull_scripts/Oct_data_pull_segregation_data.sql

```diff
@@ -0,0 +1,169 @@
+-- FETCH BY SUPERVISOR AND BRING TO MASTER, SIMILAR QUERY Needs to performed on CCS RECORD
+create unlogged table temp_thr_data_pull as select
```

Comment:
I think that these two queries could be combined to result in only one pass over `child_health_monthly`. It would require moving `age_tranche::INTEGER  BETWEEN 37 AND 72` into the case whens in the second query, but should result in less overall work and io for cluster

Also an important thing to think about in these clusters is that this is the type of query that `agg_child_health` used to use which caused OOMs. I'm not sure that we know exactly when pulling data from shards is going to increase memory, but it is something that we should keep an eye on when running this.

Could this be changed to a temporary table? temporary tables are local to a session/transaction, so it would ensure that there is nothing to clean up later